BioArchLinux/vsearch/lilac.yaml

```diff
@@ -0,0 +1,13 @@
+build_prefix: extra-x86_64
+maintainers:
+  - github: althonos
+    email: martin.larralde@embl.de
+pre_build_script: |
+  update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
+post_build_script: |
+  git_pkgbuild_commit()
+update_on:
+  - source: github
+    github: torognes/vsearch
+    use_latest_release: true
```

Comment:
Considering it's build from source, why not use_max_tag ?

Reply:
In my opinion `use_latest_release` is better, because it will not include any pre-release tags if upstream decides to release such.

Reply:
However, it depends on the custom of the repo, some repos always have the pre-release, and some repos don't release on time.